src/Features/CSharp/Portable/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -22,6 +20,11 @@ internal static class AddImportDiagnosticIds
     /// </summary>
     public const string CS0103 = nameof(CS0103);
 
+    /// <summary>
+    /// The type or namespace name 'X' does not exist in the namespace 'Y' (are you missing an assembly reference?)
+    /// </summary>
+    public const string CS0234 = nameof(CS0234);
```

Comment:
this is because you could do something like `using System.SomeNewMSNamespace` and the error will be on the inner namespace name.  This is unlike `add using for type` which has only ever worked on the left-most (unqualified name) in an expression